Sensus.Shared.iOS/Callbacks/UILocalNotifications/UILocalNotificationNotifier.cs

```diff
@@ -62,7 +48,7 @@ public override void IssueNotificationAsync(string message, string id, bool play
                         AlertBody = message,
                         TimeZone = null,  // null for UTC interpretation of FireDate
                         FireDate = DateTime.UtcNow.ToNSDate(),
-                        UserInfo = new NSDictionary(NOTIFICATION_ID_KEY, id)
```

Comment:
Without an id embedded in the notification, we won't know which one to cancel later...?

Looks like we're conflating scheduled callbacks with notifications in this PR. Probably not a good idea. Notifications do not have delays, repeats, etc. that scheduled callbacks have. I'll have to think about this.